modules/mauth-common/src/main/java/com/mdsol/mauth/util/MAuthSignatureHelper.java

```diff
@@ -51,6 +49,35 @@ public static String generateUnencryptedSignature(UUID appUUID, String httpMetho
     return httpMethod + "\n" + resourceUrl + "\n" + requestBody + "\n" + appUUID.toString() + "\n" + epochTime;
   }
 
+  /**
+   * Generate byte_arrary_to_sign for Mauth V1 protocol
+   *
+   * @deprecated
+   *   This is used for Mauth V1 protocol,
+   *   replaced by {@link #generateStringToSignV2(UUID appUUID, String httpMethod, String resourceUrl,
+   *       String queryParameters, byte[] requestBody, String epochTime)} for Mauth V2 protocol
+   *
+   * @param appUUID: app uuid
+   * @param httpMethod: Http_Verb
+   * @param resourceUrl: resource_url_path (no host, port or query string; first "/" is included)
+   * @param requestBody: request body byte[]
+   * @param epochTime: current seconds since Epoch
+   * @return byte[]
+   *   httpMethod + "\n" + resourceUrl + "\n" + requestBody + "\n" + app_uuid + "\n" + epochTime
+   *
+   */
+  @Deprecated
+  public static byte[] generateUnencryptedSignature(UUID appUUID, String httpMethod, String resourceUrl, byte[] requestBody, String epochTime) throws IOException{
+    logger.debug("Generating byte[] to sign for V1");
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    String part1 = httpMethod + "\n" + resourceUrl + "\n";
+    String part2 =  "\n" + appUUID.toString() + "\n" + epochTime;
+    baos.write(part1.getBytes());
```

Comment:
When converting String to byte[], you should always specify the encoding

Reply:
fixed, use UTF_8